src/published-data/dto/published-data.obsolete.dto.ts

```diff
@@ -23,6 +79,10 @@ export class PublishedDataObsoleteDto {
   })
   @IsString()
   @IsOptional()
+  @Expose()
```

Comment:
Do we really need to use `@Expose` in this file?

If Im not mistaken `@Expose` should only be used when `excludeExtraneousValues` is enabled or if the class is decorated with `@Exclude` to specify which field to expose within the output